### DISCUSSION TITLE
Doc comment on Create a Weaviate instance (/developers/academy/py/starter_text_data/setup_weaviate/create_instance)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1887] [Docs/blog comments]
# Doc comment on Create a Weaviate instance (/developers/academy/py/starter_text_data/setup_weaviate/create_instance)

For this unit, you can choose to create a Weaviate Cloud Service (WCS) instance or a local Docker instance.

https://weaviate.io/developers/academy/py/starter_text_data/setup_weaviate/create_instance

**Comment by homanmirgolbabaee** [comment]
✅
